packages/composables/package.json

```diff
@@ -22,8 +22,7 @@
 	},
 	"main": "dist/index.js",
 	"files": [
-		"dist",
-		"!**/*.d.ts?(.map)"
+		"dist"
```

Comment:
Do we need to add `!**/*.test.js` similar to the entry in api:

https://github.com/directus/directus/blob/c301cfbc82d1ce3ba8dad56800552c4899895b6e/api/package.json#L62

to ignore the built `use-items.test.js` test file? or maybe even another additional `!**/*.test.d.ts` if I'm not mistaken.

Reply:
let's do this separately if deemed necessary down the line.

Reply:
Update: this is now addressed in #18211!